exporter/awsxrayexporter/awsxray.go

```diff
@@ -52,10 +52,14 @@ func newTracesExporter(
 	if err != nil {
 		return nil, err
 	}
+	awsSDKV2Config, err := awsutil.GetAWSConfig(logger, cn, &cfg.AWSSessionSettings)
```

Comment:
Do we still need the `awsutil.GetAWSConfigSession` call?

Reply:
It is needed to create the xray client.